submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/mapping/extended/to/ToExtAttribute.kt

```diff
@@ -0,0 +1,13 @@
+package ac.uk.ebi.biostd.persistence.mapping.extended.to
+
+import ac.uk.ebi.biostd.persistence.model.Attribute
+import ac.uk.ebi.biostd.persistence.model.AttributeDetail
+import ebi.ac.uk.extended.model.ExtAttribute
+import ebi.ac.uk.extended.model.ExtAttributeDetail
+
+internal const val TO_EXT_ATTRIBUTE_EXTENSIONS = "ac.uk.ebi.biostd.persistence.mapping.extended.to.ToExtAttributeKt"
+
+internal fun Attribute.toExtAttribute(): ExtAttribute = ExtAttribute(
```

Comment:
As a standard, I've noticed we've been using the extended class as the file name. For example, AttributeExt.kt

Reply:
problem is that we have a different extension for the same class . Extension to convert to one way and extension to convert to the other side so mix them in single file looks pretty complicated.